modules/ebi-metagenomics/extractcoords/meta.yml

```diff
@@ -0,0 +1,58 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "extractcoords"
+description: Process output from easel-sfetch to extract SSU and LSU sequences
+keywords:
+  - easel
+  - rrna
+  - ssu
+  - lsu
+tools:
+  - "extractcoords":
+      description: "Process output from easel-sfetch to extract SSU and LSU sequences, uses mgnify-pipelines-toolkit as conda environment"
+      homepage: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      documentation: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      tool_dev_url: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      licence: ["Apache-2.0"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - easel_coords_fasta:
+      type: file
+      description: Fasta file output from running esl-sfetch to extract sequences by name
+      pattern: "*.fasta"
+  - matched_seqs_with_coords:
+      type: file
+      description: cmsearchdeoverlap output formatted for use by esl-sfetch
+      pattern: "*.matched_seqs_with_coords"
```

Comment:
Is this the actual output format of esl-esfetch? Anything more standardised?

Reply:
It's a space separated file, not tab or comma.. I guess I could use .txt in that case?